vantage/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f9d211f9-8cd2-4bab-9718-462c5a3fa9ad",
+  "app_id": "vantage",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Import your Datadog costs and track them alongside other infrastructure spend",
+    "title": "Vantage",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog Monthly Costs with Budget",
+        "image_url": "assets/images/vantage-datadog-budget-forecast.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Costs - Grouped by Service",
+        "image_url": "assets/images/vantage-datadog-grouped-report.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Provider Comparison Overview",
+        "image_url": "assets/images/vantage-datadog-provider-summary.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Category::Cost Management",
+      "Category::Cloud"
+      
+    ]
+  },
+  "author": {
+    "support_email": "support@vantage.sh",
+    "sales_email": "sales@vantage.sh"
+  },
+  "legal_terms": {
+    "eula": "https://www.vantage.sh/terms-of-service"
```

Comment:
Could you please download the eula file, upload it to this PR, and then link to that version of the document?